packages/compass-e2e-tests/installers/helpers.ts

```diff
@@ -41,5 +47,11 @@ export function execute(
         }
       }
     });
+
+    // Exit child process if main process exits
```

Comment:
Copied from https://github.com/kraenhansen/bufout which we should probably just use.